packages/orchestration/src/examples/sendAnywhere.contract.js

```diff
@@ -1,21 +1,19 @@
 import { makeStateRecord } from '@agoric/async-flow';
 import { AmountShape } from '@agoric/ertp';
-import { heapVowE } from '@agoric/vow/vat.js';
 import { InvitationShape } from '@agoric/zoe/src/typeGuards.js';
 import { Fail } from '@endo/errors';
 import { E } from '@endo/far';
 import { M } from '@endo/patterns';
-import { CosmosChainInfoShape } from '../typeGuards.js';
 import { withOrchestration } from '../utils/start-helper.js';
 import { orchestrationFns } from './sendAnywhereFlows.js';
+import { prepareChainHubAdmin } from './shared/chain-hub-cf.js';
```

Comment:
filename should be be `chain-hub-admin.js`. also consider nesting it in an `exos` dir.